dependencies.js

```diff
@@ -0,0 +1,412 @@
+"use strict";
+
+/**
+ * @typedef {Object<string, ComponentCategory>} Components
+ * @typedef {Object<string, ComponentEntry | string>} ComponentCategory
+ *
+ * @typedef ComponentEntry
+ * @property {string} [title] The title of the component.
+ * @property {string} [owner] The GitHub user name of the owner.
+ * @property {boolean} [noCSS=false] Whether the component doesn't have style sheets which should also be loaded.
+ * @property {string | string[]} [alias] An optional list of aliases for the id of the component.
+ * @property {Object<string, string>} [aliasTitles] An optional map from an alias to its title.
+ *
+ * Aliases which are not in this map will the get title of the component.
+ * @property {string | string[]} [require]
+ * @property {string | string[]} [modify]
+ * @property {string | string[]} [after]
+ */
+
+var getLoad = (function () {
```

Comment:
Definitely need to bikeshed this function name lol. Maybe `getRequiredLanguages`? Something else?

Reply:
The thing is: This function doesn't only return the required languages. It also returns all languages which have to the reloaded. Because of the reloads, even something as generic as `getDependencies` isn't a fitting name.

Also, this function can also be used for plugins.

TIL: [Bikeshedding](https://en.wiktionary.org/wiki/bikeshedding) is a thing.

Reply:
Maybe `getLoader`? Then the type of the returned object is a `Loader`. Then we'd have `Loader#load()` & `Loader#getIds()`. Or maybe `#loadIds()` for consistency? Whatever you like better.

Reply:
Really like it.
But I'll keep it `getIds()` because it doesn't load anything. (I know that you want the "load" in "loadIds" to refer to the "load" function but I (and prob anybody else) read this as a verb.) 

Maybe `getLoadIds()` ?

Reply:
`getIds` is good, let's leave it.